src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Solicitor.java

```diff
@@ -108,6 +108,14 @@ public boolean hasOrgId() {
         return false;
     }
 
+    @JsonIgnore
```

Comment:
thinking about it, that isn't nicely readable.
maybe something like 
 if (organisationPolicy == null || organisationPolicy.getOrganisation() == null) {
        return false;
    }
    String orgName = organisationPolicy.getOrganisation().getOrganisationName();
    return orgName != null && !orgName.isEmpty();

Reply:
I made this consistent with the existing method (`hasOrgID`) which has been in place for 3 years since Linus added it, so that someone looking at the two methods can understand quickly that they do the same thing. Although I considered refactoring the old method, I felt the improvements in readability seemed minor/subjective and it was simple enough to understand, so it wasn't worth refactoring working code

Reply:
fair enough